src/site/layouts/campaign_landing_page.drupal.liquid

```diff
@@ -653,5 +653,34 @@
   </main>
 </div>
 
+<!-- TEMPORARY -->
+<!-- =============== -->
+<script>
+  const checkCLPAuth = () => {
+    const password = prompt('This page is in BETA. Please enter the password to access it:');
+
+    // Escape early if the password is incorrect.
+    if (password !== 'clptest123') {
+      checkCLPAuth();
+      return;
+    }
+
+    // Derive the content element.
+    const content = document.getElementById('content');
+
+    // Escape early if we cannot find it.
+    if (!content) {
+      console.error('Unable to find #content to show.');
+      return;
+    }
+
+    // Show the content.
+    content.classList.remove('vads-u-display--none');
+  }
+
+  checkCLPAuth();
+</script>
```

Comment:
Unfortunately we have to use ES5 flavor like `var` so that our stakeholder using IE are prompted! 😬

Reply:
Ah makes sense! Will switch 👍

Reply:
Updated!

Reply:
`. classList` too unfortunately, sorry!

Reply:
oh true! `classList` isn't supported in IE9 and below, but is supported in higher versions. I switched it to use `.className` instead